### PR TITLE
v1.5.0

### PR DESCRIPTION
**Pull request by rix0rrr**
See CHANGELOG

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
+# [1.5.0](https://github.com/aws/aws-cdk/compare/v1.4.0...v1.5.0) (2019-08-20)
```
Comment (RomainMuller) [inline CHANGELOG.md]: You'll want to be fixing the heading here - the top-level heading has to be moved back up here & this needs to be H2.

Comment (rix0rrr) [reply CHANGELOG.md]: Why does this keep happening???

**Comment by jogold** [comment]
@rix0rrr have you seen my comment on #3677 : https://github.com/aws/aws-cdk/pull/3677#discussion_r314625719 regarding the casing for `notactions` and `notresources`? Just wanted you to know about it before it gets released, breaking changes etc...

File: CHANGELOG.md (hunk 1 above)
Comment (RomainMuller) [reply CHANGELOG.md]: I dunno 🤷🏻‍♂️ but I'll gladly agree it's a PITA. Maybe file an issue about it so we can prioritize looking into?

**Comment by RomainMuller** [comment]
Tagged `do-not-merge` to make sure @jogold comment gets looked at before this goes in.

**Comment by rix0rrr** [comment]
You are right, jogold@, thanks for being so on top of this.

This PR is blocked by https://github.com/aws/aws-cdk/pull/3727.

**Comment by rix0rrr** [comment]
Semantic PR check seems to be broken :/

**Pull request merged**